docs/services/MicrosoftGraph.md

```diff
@@ -75,6 +75,15 @@ if (!await MicrosoftGraphService.Instance.LoginAsync())
  return;
 }
 ```
+```csharp
```

Comment:
The csharp and vs code snippets (above and below) need to go together. Can you move this new snippet in it's own section (just like all the other ones)?

Reply:
Yep , sure